### ISSUE TITLE
BUG: Unable to download KVIrc, browsers have dropped FTP support

### ISSUE THREAD

**Issue by mjevans** [kvirc/KVIrc#2516]
Unable to download KVIrc, browsers have dropped FTP support

Chrome / MS Edge (since it's based on Chrome now)
2021 Jan https://developers.google.com/web/updates/2020/12/chrome-88-deps-rems
https://docs.microsoft.com/en-us/microsoft-edge/web-platform/site-impacting-changes

Firefox:
2021 April https://www.mozilla.org/en-US/firefox/88.0/releasenotes/
Why https://blog.mozilla.org/security/2021/07/20/stopping-ftp-support-in-firefox-90/

The TL;DR of why appears to be a push to get everything on HTTPS to help prevent man in the middle attacks.

**Comment by mjevans** [comment]
One possible short term solution is to use Github to host the compiled releases.

https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

**Comment by wodim** [comment]
We have been recommending users to use nightly versions for a while.

https://github.com/kvirc/KVIrc/wiki/Downloading-KVIrcs-nightly-source-or-binaries

**Comment by ctrlaltca** [comment]
We are about to release a new version, please try https://github.com/kvirc/KVIrc/releases/tag/5.2.0-beta1-fix and report if this issue is still present. Thank you
